src/sparsezoo/analyze/analysis.py

```diff
@@ -1305,6 +1305,7 @@ def create(cls, file_path: Union[str, ModelProto]) -> "ModelAnalysis":
             result = ModelAnalysis.from_onnx(Path(file_path) / "model.onnx")
 
         elif file_path.startswith("zoo:"):
+            Model(file_path).deployment.download()
```

Comment:
Why not use .path to optionally download

Reply:
Addressed! Good catch